helm-charts/infisical/README.md

```diff
@@ -0,0 +1,187 @@
+| `mongodbConnection.externalMongoDBConnectionString` | External MongoDB connection string                                                                                                                                                        | `""`                 |
+
+
+### Ingress parameters
+
+| Name               | Description                                 | Value             |
+| ------------------ | ------------------------------------------- | ----------------- |
+| `ingress.enabled`  | Enable ingress                              | `true`            |
+| `ingress.hostName` | Ingress hostname (your custom domain name)  | `infisical.local` |
+| `ingress.tls`      | Ingress TLS hosts (matching above hostName) | `[]`              |
+
+
+### Mailhog parameters
+
+| Name                               | Description                | Value                     |
+| ---------------------------------- | -------------------------- | ------------------------- |
+| `mailhog.enabled`                  | Enable Mailhog             | `false`                   |
+| `mailhog.fullnameOverride`         | Fullname override          | `mailhog`                 |
+| `mailhog.nameOverride`             | Name override              | `""`                      |
+| `mailhog.image.repository`         | Image repository           | `lytrax/mailhog`          |
+| `mailhog.image.tag`                | Image tag                  | `latest`                  |
+| `mailhog.image.pullPolicy`         | Image pull policy          | `IfNotPresent`            |
+| `mailhog.containerPort.http.port`  | Mailhog HTTP port (Web UI) | `8025`                    |
+| `mailhog.containerPort.smtp.port`  | Mailhog SMTP port (Mail)   | `1025`                    |
+| `mailhog.ingress.enabled`          | Enable ingress             | `true`                    |
+| `mailhog.ingress.ingressClassName` | Ingress class name         | `nginx`                   |
+| `mailhog.ingress.annotations`      | Ingress annotations        | `{}`                      |
+| `mailhog.ingress.labels`           | Ingress labels             | `{}`                      |
+| `mailhog.ingress.hosts[0].host`    | Mailhog host               | `mailhog.infisical.local` |
+
+
+## Persistence
+
+The database persistence is enabled by default, your volumes will remain on your cluster even after uninstalling the chart. To disable persistence, set this value `mongodb.persistence.enabled: false`
+
+## Local development
+
+Use below values if you want to setup a local development environment, and adapt those variables as you need. Below example will deploy the following :
+- https://infisical.local
+  - Your local Infisical instance
+  - You may have to add `infisical.local` to your `/etc/hosts` or similar depending your OS
```

Comment:
Do you think people will know to point their ingress external IP to this domain in the /etc/hosts or should we explain it?

Reply:
I don't think that is always the case because when I installed ingress with my minikube, the ingress IP I had to point to my domain in /etc.hosts was a private IP `192.168.49.2`